packages/twenty-front/src/modules/information-banner/components/billing/InformationBannerBillingSubscriptionPaused.tsx

```diff
@@ -0,0 +1,39 @@
+import { InformationBanner } from '@/information-banner/components/InformationBanner';
+import { AppPath } from '@/types/AppPath';
+import { SettingsPath } from '@/types/SettingsPath';
+import { useSubscriptionStatus } from '@/workspace/hooks/useSubscriptionStatus';
+import { isDefined } from 'twenty-ui';
+import {
+  SubscriptionStatus,
+  useBillingPortalSessionQuery,
+} from '~/generated/graphql';
+
+export const InformationBannerBillingSubscriptionPaused = () => {
+  const subscriptionStatus = useSubscriptionStatus();
+
+  const { data, loading } = useBillingPortalSessionQuery({
```

Comment:
I think this will make always make an additional query. I feel no query should be made if the subscription is not paused. To fix it: either we need to add a subcomponent in InformationBannerBillingSubscriptionPaused.tsx that will only be loaded if the activationStatus is SUSPENDED, or add the SUSPENDED condition in the parent InformationBannerWrapper component